pulp_ansible/app/galaxy/views.py

```diff
@@ -157,8 +157,10 @@ def post(self, request, path):
         artifact = Artifact.init_and_validate(serializer.validated_data["file"])
         artifact.save()
 
+        filename = serializer.validated_data["file"].name
```

Comment:
One of the concerns is about falsified content so we can't rely on this user provided data. We can rely that the name should always be `namespace-name-version.tar.gz` like the example here https://docs.ansible.com/ansible/latest/user_guide/collections_using.html#id2

Maybe we need another helper function like this one? https://github.com/pulp/pulp_ansible/blob/master/pulp_ansible/app/models.py#L166 but for the filename generation.

Reply:
the extension will always be `tar.gz`?